whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -775,8 +776,7 @@ class PostgreSQLComponent {
                         throw new ConflictingHoldException("Already exists a holding record for ${heldBy} and bib: $holdingFor")
                 }
 
-                if (linkFinder != null)
-                    linkFinder.normalizeIdentifiers(doc)
```

Comment:
Why has this call been removed?

Reply:
This call is redundant, see https://github.com/libris/librisxl/blob/23315155947f69969f6f352f678071421b0a461b/whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy#L741 further up.

Reply:
Ok, see https://github.com/libris/librisxl/pull/1552/commits/cfa88469d6615a9a9666fbef9deabcfdf0d0603c

`normalizeIdentifiers` is already called `normalizeDocumentForStorage`
https://github.com/libris/librisxl/blob/23315155947f69969f6f352f678071421b0a461b/whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy#L741